x/stake/params.go

```diff
@@ -0,0 +1,36 @@
+package stake
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Params defines the high level settings for staking
+type Params struct {
```

Comment:
this will move to fees module ?

Reply:
Not all of Params

I've been mulling over whether inflation atom creation belongs in staking or fees. leaning towards staking, fees will need a way to receive the provisions from staking, which can be a part of the fee-distribution interface type in the std lib. So in that case everything would actually stay from params